staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -57,9 +57,10 @@ func GetTransformerOverrides(filepath string) (map[schema.GroupResource]value.Tr
 
 // ParseEncryptionConfiguration parses configuration data and returns the transformer overrides
 func ParseEncryptionConfiguration(f io.Reader) (map[schema.GroupResource]value.Transformer, error) {
-	configFileContents, err := ioutil.ReadAll(f)
+	// Upper limit of 100 KB on file size.
+	configFileContents, err := readUptoLength(f, 102400)
```

Comment:
I wonder if this size should be configurable with some sort of environment flag. Also perhaps we shouldn't hardcode that value, maybe use a default.